npm/test-integration.js

```diff
@@ -61,8 +61,7 @@ module.exports = function (exit) {
                     collection: test.collectionJSON,
                     environment: test.environmentJSON,
                     globals: test.globalsJSON,
-                    iterationData: test.dataCSV || test.dataJSON,
-                    abortOnError: true
```

Comment:
Also - does stop on failure store the last error on which it stopped and pass it to summary.error?